frontend/playwright.enterprise.config.ts

```diff
@@ -23,9 +24,9 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 4 : undefined,
+  workers: process.env.CI ? 8 : undefined,
```

Comment:
Let's remove the comment `/* Opt out of parallel tests on CI. */`, it's incorrect